src/Compiler/Driver/ParseAndCheckInputs.fs

```diff
@@ -1961,7 +1961,54 @@ let CheckMultipleInputsUsingGraphMode
 
         partialResults, tcState)
 
+let TryReuseTypecheckingResults (tcConfig: TcConfig) inputs =
+    let tcDataFileName = FileSystem.GetFullPathShim "FSharpTypecheckingData"
```

Comment:
This should probably come from msbuild, via the value of the flag, and should be set to an intermediate output

Reply:
Yeah, I need to resolve and set the proper location for the file here.

The name got inspiration for `FSharpSignatureData` and `FSharpOptimizationData` we already have, but it can be whatever.

Reply:
@vzarytovskii just so that I don't reinvent the wheel here - do you know if we already have a way to extract msbuild vars somewhere in the compiler?

Reply:
Okay so we discussed this with Kevin, `outputDir` should be enough here.